recipes/date/all/test_v1_package/test_package.cpp

```diff
@@ -0,0 +1,62 @@
+#include "date/date.h"
```

Comment:
This file can be removed, in case you consume directly from test_package instead.